pkg/store/storepb/rpc.proto

```diff
@@ -122,6 +122,10 @@ message SeriesRequest {
   // shard_info is used by the querier to request a specific
   // shard of blocks instead of entire blocks.
   ShardInfo shard_info = 13;
+
+  // replica_labels are labels used for deduplicating
+  // series during querying.
+  repeated string replica_labels = 14;
```

Comment:
```suggestion
  // sort_without_labels represents labels that we want in the response series, but we want to move them to the end of series (sort by all but them). This is useful for callers to perform efficient deduplication on higher levels.
  repeated string sort_without_labels = 14;
```

Reply:
replica label means nothing to store API implementations.

Reply:
Thanks, updated.